api/pom.xml

```diff
@@ -16,19 +16,19 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-annotations</artifactId>
-            <version>2.9.2</version>
+            <version>2.12.5</version>
```

Comment:
Why there are here specific dependency versions on the pom.xml?
To give context, on another project which uses Spring we just defined a specific spring version on the parent, and based on it, all of the dependencies will use the version compatible with the spring

Reply:
Maybe jackson-annotation is not included in our Spring dependencies. In this case it would make sense to use version numbers. But it could also be possible, that this package is included by some Spring dependency and that the version number is redundant.

However, this version number was not introduced by me, so I think this discussion doesn't belong to this PR. But Maybe we should create a different issue to clean up our dependencies.

Reply:
This suggestion is not related directly to the issue of this PR so based on it this conversation is solved